app/grandchallenge/verifications/models.py

```diff
@@ -80,6 +83,29 @@ def save(self, *args, **kwargs):
 
         super().save(*args, **kwargs)
 
+        if self.has_changed("is_verified") and self.is_verified:
+            permission_request_classes = [
```

Comment:
The contents of block should be in its own method. See L109-L110.

Reply:
Done

Reply:
No it isn't.